bokehjs/src/coffee/renderer/guide/categorical_axis.coffee

```diff
@@ -13,6 +9,7 @@ class CategoricalAxis extends Axis.Model
 
   initialize: (attrs, objects) ->
     super(attrs, objects)
+    Collections = require("../../common/base").Collections
```

Comment:
I moved it to make it work. `.Collections` is undefined if I put it at the top. Not entirely sure why. It would be nice to at least understand, though. @tswicegood any ideas?